components/layout_2020/flow/root.rs

```diff
@@ -53,10 +59,14 @@ pub struct FragmentTree {
     root_fragments: Vec<ArcRefCell<Fragment>>,
 
     /// The scrollable overflow of the root of the fragment tree.
```

Comment:
This comment might need a small update to reflect the fact that this is for all of the root fragments.